frontend/src/components/StatusTag.vue

```diff
@@ -197,12 +193,15 @@ export default {
       }
       return 'primary'
     },
+    textColor () {
+      if (this.isError) {
+        return 'white'
+      }
+      return this.color
+    },
     visible () {
       if (!this.isAdmin) {
-        const { type } = this.condition
-
-        const conditionMetadata = this.conditionMetadataFromType(type)
-        return !get(conditionMetadata, 'showAdminOnly', false)
+        return !get(this.condition, 'showAdminOnly', false)
```

Comment:
this does not work anymore. Now, as non-admin I can see the control plane chip

Reply:
fixed